module/metrics.go

```diff
@@ -498,6 +498,8 @@ type ComplianceMetrics interface {
 	EpochFallbackModeTriggered()
 	// EpochFallbackModeExited reports that EFM is no longer triggered.
 	EpochFallbackModeExited()
+	// ProtocolKVStoreVersion reports the latest finalized protocol state KVStore version.
+	ProtocolKVStoreVersion(version uint64)
```

Comment:
Maybe I am oversimplifying it, but in my head, the KV store is essentially an implementation detail of the Protocol State to store different parameters. I find it very unlikely that we are going to version the protocol state beyond the version we already have for the KV store. Therefore, I would be inclined to just call this the "ProtocolStateVersion"
```suggestion
	// ProtocolStateVersion reports the version of the latest finalized protocol state.
	ProtocolStateVersion(version uint64)
```